app/src/main/java/org/simple/clinic/home/overdue/appointmentreminder/AppointmentReminderSheet.kt

```diff
@@ -110,6 +111,19 @@ class AppointmentReminderSheet : BottomSheetActivity() {
     decrementDateButton.isEnabled = state
   }
 
+  private fun textForAppointmentReminder(timeAmount: Int, chronoUnit: ChronoUnit): String {
+    val quantityStringResourceId = when (chronoUnit) {
+      ChronoUnit.DAYS -> {
+        R.plurals.appointmentreminder_days
+      }
+      ChronoUnit.WEEKS -> {
+        R.plurals.appointmentreminder_weeks
+      }
+      else -> throw IllegalArgumentException("$chronoUnit is unsupported type for appointment reminders")
+    }
+    return resources.getQuantityString(quantityStringResourceId, timeAmount, timeAmount)
```

Comment:
This might cause a crash since we are passing `Int` values to the format string while it expects `String`. Could you please test this out and verify that it renders as expected?

Reply:
I have tested it on my phone, it's working. I will also try it on API 21 and see if it crashes.